packages/accordions/README.md

```diff
@@ -0,0 +1,47 @@
+# @zendeskgarden/react-accordions [![npm version](https://img.shields.io/npm/v/@zendeskgarden/react-accordions.svg?style=flat-square)](https://www.npmjs.com/package/@zendeskgarden/react-accordions)
+
+This package includes components related to accordions in the
+[Garden Design System](https://zendeskgarden.github.io/).
+
+## Installation
+
+```sh
+npm install @zendeskgarden/react-accordions
+
+# Peer Dependencies - Also Required
+npm install react react-dom prop-types styled-components @zendeskgarden/react-theming
+```
+
+## Usage
```

Comment:
This `Usage` section could be structured like the following when the Accordion component is introduced:

```
## Accordion Usage
<Accordion />

## Stepper Usage
<Stepper />
```